apps/parsley/src/components/PopoverButton/index.tsx

```diff
@@ -10,22 +10,30 @@ interface PopoverButtonProps extends Omit<ButtonProps, "children"> {
   onClick?: () => void;
   buttonText: React.ReactNode | string;
   buttonRef: RefObject<HTMLButtonElement>;
+  disableOutsideClick?: boolean;
+  isOpen: boolean;
+  setIsOpen: (value: boolean) => void;
 }
 
 const PopoverButton: React.FC<PopoverButtonProps> = ({
   buttonRef,
   buttonText,
   children,
+  disableOutsideClick,
```

Comment:
It's very easy to close the menu when outside click is enabled and I want to make sure users get to see the introductory guide cues.